### ISSUE TITLE
Qtip scrolls with page scroll in Chrome

### ISSUE THREAD

**Issue by emik** [qTip2/qTip2#689]
In Chrome, the combination of overflow:auto/scroll and position:relative/absolute on the body results in qtip scrolling with the page, becoming detached from its target element. It works fine in Firefox and IE.

This jsfiddle example is fixed by removing either the position or the overflow property: http://jsfiddle.net/bQ6Fv/1/

**Comment by jhenderson22** [comment]
I am experiencing the same issue. What is the solution / work-around?

**Comment by Snesi** [comment]
Yep, same problem here :weary:. The problem is I need overflow: auto to avoid margin collapse.

**Comment by Snesi** [comment]
I just realised that this also happens in Firefox and Safari. :weary: 
Isn't anyone else annoyed by this??

**Comment by Snesi** [comment]
Hi!
I just discovered that by commenting https://github.com/qTip2/qTip2/blob/master/src/core/position.js#L221 and https://github.com/qTip2/qTip2/blob/master/src/core/position.js#L222
the issue is fixed. 

@Craga89 what is the scroll function in `resposition.offset` for??

```javascript
	function scroll(e, i) {
		pos.left += i * e.scrollLeft();
		pos.top += i * e.scrollTop();
	}
```

**Comment by Snesi** [comment]
Created a provisional fix to this problem here https://github.com/Snesi/qTip2.git.

But I'm not going to create a pull request until I fully understand the problem.

**Comment by Snesi** [comment]
@Craga89 If I have time during the weekend I'll try to fix this issue and make a pull request. But I'm starting to think that my problem may be an isolated thing. I don't know if other people have the same bug, because my qtips scroll in all Browsers, and even with the "fix" I proposed, the qtips are still scrolling in iOS.

**Comment by kfrancis** [comment]
Seeing the same issue here.

**Comment by davosmith** [comment]
This still seems to be a problem - I created a JSFiddle ( https://jsfiddle.net/davosmith/4t9r7oxj/ ) before I realised others had already reported this.

I'd suggest either the code that sets: scrolled = $(parent) needs to ignore 'body' tags, or the code below it that checks to see if scrolled[0] !== ownerDocument[0] should also check that this isn't the 'body' tag.

**Comment by sinrise** [comment]
This is still happening for me. I just grabbed v3.0.3

**Comment by hxiaope** [comment]
press F12 then tips not follow element
